libs/domain/auth/login/src/login.styles.ts

```diff
@@ -1,13 +1,18 @@
 import { css } from 'lit';
 
 export const styles = css`
-  oryx-card form > * {
-    margin: 10px 0;
+  form {
+    display: flex;
+    flex-direction: column;
+    gap: 18px;
   }
 
   .options {
     display: flex;
     align-items: center;
-    justify-content: space-between;
+  }
+
+  .options .forgot-password {
```

Comment:
```suggestion
  oryx-link {
```
and drop the class from the oryx-link

Reply:
Not too sure about this, we might change link to button again going forward.

Reply:
I believe that this will be a case when we make this only, because it can require additional stylisation